core/app/models/spree/promotion/actions/create_item_adjustments.rb

```diff
@@ -15,6 +15,19 @@ class CreateItemAdjustments < PromotionAction
         before_destroy :remove_adjustments_from_incomplete_orders
         before_discard :remove_adjustments_from_incomplete_orders
 
+        def can_discount?(klass)
```

Comment:
Yes, I think we should declare it on the `Spree::PromotionAction` base class and make it return `false` by default. This will also be the correct response for the `CreateAdjustment` class, which IMO should be deprecated and replaced by `CreateItemAdjustments` with the `DistributedAmount` calculator.